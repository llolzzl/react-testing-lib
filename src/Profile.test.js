import React from "react";
import { render } from "@testing-library/react";
import Profile from "./Profile";

describe("<Profile />", () => {
    it("matches snapshot", () => {
        const utils = render(<Profile username="hwangj9642" name="황진주" />);
        expect(utils.container).toMatchSnapshot();
    });
    it("shows thhe props correctly", () => {
        const utils = render(<Profile username="hwangj9642" name="황진주" />);
        utils.getByText("hwangj96421");
        utils.getByText("(황진주)");
        // utils.getByText("hwangj9642 (황진주)");
    });
});