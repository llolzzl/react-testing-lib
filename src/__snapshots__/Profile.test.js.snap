// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Profile /> matches snapshot 1`] = `
<div>
  <div>
    <b>
      hwangj9642
      1
    </b>
     
    <span>
      (
      황진주
      )
    </span>
  </div>
</div>
`;
